package.json

```diff
@@ -12,142 +12,126 @@
     "ember-cli-update": "0.22.0",
-    "ember-composability-tools": "0.0.11",
+    "ember-composability-tools": "^0.0.12",
     "ember-composable-helpers": "3.2.0",
-    "ember-concurrency": "1.0.0",
-    "ember-concurrency-decorators": "1.0.0-beta.4",
-    "ember-concurrency-retryable": "0.6.0",
+    "ember-concurrency": "^2.3.7",
+    "ember-concurrency-retryable": "1.0.0",
     "ember-content-placeholders": "^1.0.0",
-    "ember-data": "~3.19.0",
+    "ember-data": "~3.28.0",
     "ember-data-model-fragments": "5.0.0-beta.8",
     "ember-decorators": "6.0.0",
     "ember-decorators-polyfill": "1.0.5",
     "ember-element-resize-detector": "0.2.1",
     "ember-export-application-global": "^2.0.1",
-    "ember-fetch": "^5.1.3",
+    "ember-fetch": "^8.1.1",
     "ember-g-recaptcha": "^1.3.0",
-    "ember-in-viewport": "^3.1.3",
-    "ember-load-initializers": "^2.1.1",
+    "ember-load-initializers": "^2.1.2",
     "ember-local-storage": "2.0.2",
-    "ember-mapbox-composer": "0.4.0",
-    "ember-mapbox-gl": "0.16.0",
-    "ember-mapbox-gl-draw": "2.0.0",
+    "ember-mapbox-gl": "^1.0.0",
+    "ember-mapbox-gl-draw": "https://github.com/allthesignals/ember-mapbox-gl-draw#patch-3",
     "ember-maybe-import-regenerator": "^0.1.6",
     "ember-metrics": "0.13.0",
     "ember-native-dom-helpers": "^0.6.2",
-    "ember-needs-async": "git+https://github.com/NYCPlanning/ember-needs-async#4133bcfffe478f71845d1a9147fee25c1fdb2038",
     "ember-parachute": "1.0.2",
-    "ember-power-select": "^2.0.4",
-    "ember-promise-helpers": "^1.0.6",
-    "ember-query-params-service": "0.4.0",
-    "ember-qunit": "4.6.0",
-    "ember-radio-button": "1.2.4",
-    "ember-resolver": "^8.0.0",
+    "ember-promise-helpers": "^1.0.9",
+    "ember-qunit": "^5.1.4",
+    "ember-resolver": "^8.0.2",
     "ember-responsive": "^5.0.0",
-    "ember-source": "~3.19.0",
-    "ember-template-lint": "^2.8.0",
-    "ember-test-helpers": "^0.6.3",
+    "ember-source": "~3.28.11",
+    "ember-template-lint": "^3.6.0",
     "ember-test-selectors": "2.0.0",
-    "ember-tether": "v1.0.0-beta.3",
     "ember-tooltips": "^3.1.0",
-    "ember-truth-helpers": "^2.1.0",
+    "ember-truth-helpers": "^3.1.1",
     "ember-unused-components": "0.2.0",
     "ember-window-mock": "0.5.4",
     "ember-wormhole": "^0.5.2",
-    "eslint": "^7.1.0",
+    "eslint": "^7.32.0",
     "eslint-config-airbnb-base": "^13.0.0",
-    "eslint-plugin-ember": "^8.6.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-ember": "^10.5.4",
     "eslint-plugin-ember-best-practices": "1.1.1",
     "eslint-plugin-import": "^2.12.0",
     "eslint-plugin-node": "^11.1.0",
-    "fastboot": "2.0.1",
-    "foundation-sites": "6.5.0-rc.4",
+    "eslint-plugin-prettier": "^3.4.1",
+    "eslint-plugin-qunit": "^6.2.0",
+    "foundation-sites": "^6.7.5",
     "husky": "^6.0.0",
-    "labs-ember-search": "3.1.6",
-    "labs-ui": "^0.0.24",
+    "labs-shared": "https://github.com/allthesignals/labs-shared#ember-3.28",
```

Comment:
Should we update this reference to point to the [NYCPlanning fork](https://github.com/NYCPlanning/ose-ember-addons)?